mmv1/third_party/terraform/data_sources/data_source_google_cloudbuild_trigger.go

```diff
@@ -0,0 +1,46 @@
+package google
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceGoogleCloudBuildTrigger() *schema.Resource {
+
+	dsSchema := datasourceSchemaFromResourceSchema(resourceCloudBuildTrigger().Schema)
+
+	addRequiredFieldsToSchema(dsSchema, "trigger_id")
+	addOptionalFieldsToSchema(dsSchema, "project")
+
+	//adding location field explicitly in schema because it requires to set default value for it.
+	dsSchema["location"] = &schema.Schema{
```

Comment:
I think it would make more sense to mark location as required and remove this default.

Location only has a global default due to how the resource was originally implemented, it would be preferable to just have it required